test/float_tex.cpp

```diff
@@ -9,10 +9,9 @@
 #include <cmath>
 #include <iostream>
 #include <vector>
-extern "C" {
```

Comment:
A lot of these changes look unrelated?

Reply:
Yeah, I just cleaned up the test a little while I was working on it.